### PR TITLE
Add action-shellcheck to lint shell scripts

### LINKED ISSUE #17: Add action-shellcheck
Add `action-shellcheck` to lint shell scripts.

Link:
https://github.com/reviewdog/action-shellcheck

### PR DESCRIPTION
**Pull request by vvkpd**
Close #17

### CODE REVIEW

**Comment by mstruebing** [comment]
Shellcheck is a tool which can be executed like any other tool on your local machine or in a CI/CD pipeline.

Here is the Github repository: https://github.com/koalaman/shellcheck

You can run it on your machine or even paste the code into a webpage and get the result from the linter:

https://www.shellcheck.net/

**Comment by vvkpd** [comment]
Hey @mstruebing, Thanks for you help.
I ran shell-check locally and fixed lint issues. I have pushed the  changes.
please take a look.
Thanks

File: .github/workflows/reviewdog.yml [hunk 1]
```diff
@@ -11,3 +11,13 @@ jobs:
...
+    # Use shellcheck to lint shell scripts
+    shellcheck:
+    name: runner / shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: shellcheck
+        uses: reviewdog/action-shellcheck@v1
+        with:
+          github_token: ${{ secrets.github_token }}
```
Comment (mgrachev) [inline .github/workflows/reviewdog.yml:23]: Please add new line.

File: .github/workflows/reviewdog.yml [hunk 2]
```diff
@@ -11,3 +11,13 @@ jobs:
         uses: reviewdog/action-hadolint@v1
         with:
           github_token: ${{ secrets.github_token }}
+    # Use shellcheck to lint shell scripts
+    shellcheck:
```
Comment (mgrachev) [inline .github/workflows/reviewdog.yml:15]:
```suggestion
  shellcheck:
```

**Review by mgrachev — COMMENTED** [review]
@vvkpd Thank you for your contribution! 👍

I have left some comments. Please look at them 👀

File: .github/workflows/reviewdog.yml [hunk 3]
```diff
@@ -11,3 +11,13 @@ jobs:
         uses: reviewdog/action-hadolint@v1
         with:
           github_token: ${{ secrets.github_token }}
+    # Use shellcheck to lint shell scripts
```
Comment (mgrachev) [inline .github/workflows/reviewdog.yml:14]:
```suggestion
  # Use shellcheck to lint shell scripts
```

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@vvkpd Thank you! 👍

**Pull request merged**